sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/AzureMonitorTransmitter.cs

```diff
@@ -56,84 +55,173 @@ public async ValueTask<int> TrackAsync(IEnumerable<TelemetryItem> telemetryItems
             try
             {
                 using var httpMessage = async ?
-                    await this._applicationInsightsRestClient.InternalTrackAsync(telemetryItems, cancellationToken).ConfigureAwait(false) :
-                    this._applicationInsightsRestClient.InternalTrackAsync(telemetryItems, cancellationToken).Result;
+                    await _applicationInsightsRestClient.InternalTrackAsync(telemetryItems, cancellationToken).ConfigureAwait(false) :
+                    _applicationInsightsRestClient.InternalTrackAsync(telemetryItems, cancellationToken).Result;
 
-                if (httpMessage != null)
+                result = GetRequestStatus(httpMessage);
```

Comment:
nit: As we are working on the response, should we call this one as `GetResponseStatus` or `IsSuccess`?